pkgs/by-name/hy/hyfetch/package.nix

```diff
@@ -1,23 +1,23 @@
-{ lib
-, fetchFromGitHub
-, python3
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
 }:
-
-python3.pkgs.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "hyfetch";
-  version = "1.4.11";
-  format = "setuptools";
+  version = "1.99.0";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "hykilpikonna";
     repo = "hyfetch";
     rev = "refs/tags/${version}";
-    hash = "sha256-xzN/tbS5BUvpKeozesE99gNp3NRDjvf4Qx7BHLc4svo=";
+    hash = "sha256-GL1/V+LgSXJ4b28PfinScDrJhU9VDa4pVi24zWEzbAk=";
   };
 
-  propagatedBuildInputs = with python3.pkgs; [
-    typing-extensions
+  dependencies = with python3Packages; [
     setuptools
```

Comment:
Misread and thought this was `format = "other";` for the first review, *oops*

`setuptools` [should be in `build-system`](https://nixos.org/manual/nixpkgs/stable/#buildpythonpackage-parameters)